### DISCUSSION TITLE
Contribution Guidelines

### DISCUSSION THREAD

**Discussion by rsalus** [roflmuffin/CounterStrikeSharp#599] [Q&A]
Hey there,

Just wondering where I can find the contribution guidelines. I've found [this](https://github.com/roflmuffin/CounterStrikeSharp/blob/main/CONTRIBUTING.md), but it seems more like boiler-plate and isn't specific to this project. I have a lot of experience with dotnet and would love to help out.

Thanks.

**Comment by KillStr3aK** [comment]
Hi, that guideline is correct, also you can propose or request changes either in a new issue / pull request, or in our discord server in the feature-requests forum.

Pull requests are also always reviewed and only merged if everything fits.

Thanks for your interest!
We'd love if you could contribute to this community driven project.
